account.go

```diff
@@ -528,13 +528,21 @@ func paymentXLM(from SeedStr, to AddressStr, amount, memoText string) (ledger in
 
 // PaymentXLMTransaction creates a signed transaction to send a payment from 'from' to 'to' for 'amount' lumens.
 func PaymentXLMTransaction(from SeedStr, to AddressStr, amount, memoText string,
+	seqnoProvider build.SequenceProvider, timeBounds *build.Timebounds, baseFee uint64) (res SignResult, err error) {
+	memo := NewMemoText(memoText)
+	return PaymentXLMTransactionWithMemo(from, to, amount, memo, seqnoProvider, timeBounds, baseFee)
+}
```

Comment:
would be simpler to scrap the underpowered function and have just `PaymentXLMTransaction` that takes `*Memo`

Reply:
wanted step 1 of the refactor to require as few changes as possible for whatever is calling this.